trulens_eval/trulens_eval/feedback/provider/endpoint/base.py

```diff
@@ -224,22 +225,29 @@ def _instrument_module(self, mod: ModuleType, method_name: str) -> None:
             setattr(mod, method_name, w)
 
     def _instrument_class(self, cls, method_name: str) -> None:
-        if hasattr(cls, method_name):
+        if safe_hasattr(cls, method_name):
             logger.debug(
                 f"Instrumenting {cls.__name__}.{method_name} for {self.name}"
             )
             func = getattr(cls, method_name)
             w = self.wrap_function(func)
             setattr(cls, method_name, w)
 
-    def _instrument_module_members(self, mod: ModuleType, method_name: str):
-        logger.debug(
-            f"Instrumenting {mod.__package__}.*.{method_name} for {self.name}"
-        )
+    # TODO: Remove
```

Comment:
Do we need this comment block?

Reply:
We can remove. This was a dev method we ended up not using. Nice catch.